app/models/repository_list_item.rb

```diff
@@ -0,0 +1,15 @@
+class RepositoryListItem < ApplicationRecord
+  has_many :repository_list_values,
+           foreign_key: 'selected_item_id'
```

Comment:
May be it's needed to add dependent destroy here?

Reply:
probably not

Reply:
But since there is no point in existence of repository_list_value if no item is selected we need delete all cells once one item type is deleted